datalad_registry/utils/misc.py

```diff
@@ -9,3 +13,29 @@ class StrEnum(str, Enum):
     """
 
     pass
+
+
+def allocate_ds_path() -> Path:
```

Comment:
@yarikoptic This function generates values to be assigned to `cache_path` of the `URL` model.